winterfell/forms/VBA-21-0966-ARE.json

```diff
@@ -1,4 +1,8 @@
 {
+  "unofficialTitle": "Intent to File",
```

Comment:
I agree, those official names were just a placeholder, I assumed we would go back and make them more user friendly eventually. This file is NOT autogenerated, so you are free to add fields, as long as that same field will exist for all the forms for consistency.